lib/active_merchant/billing/gateways/quickpay/quickpay_v4to7.rb

```diff
@@ -83,7 +83,7 @@ def store(creditcard, options = {})
         post = {}
 
         add_creditcard(post, creditcard, options)
-        add_amount(post, 0, options) if @protocol >= 7
```

Comment:
Okay.  Ideally we would _just_ send the currency, and not the amount, for v3, but this seems to be the quickest way to get the currency in there.